### PR TITLE
[WFLY-16916] Wrong description of enable-graceful-txn-shutdown attribute

### PR DESCRIPTION
**Pull request by petrberan**
https://issues.redhat.com/browse/WFLY-16916

### CODE REVIEW

File: ejb3/src/main/resources/org/jboss/as/ejb3/subsystem/LocalDescriptions.properties
Comment (bstansberry) [inline ejb3/src/main/resources/org/jboss/as/ejb3/subsystem/LocalDescriptions.properties:27]: Further improvement:

s/omit this tag/leave this attribute undefined or set it to 'false'/g

'Tag' is a presumably reference to xml but this string isn't part of an xml schema.

Comment (petrberan) [reply ejb3/src/main/resources/org/jboss/as/ejb3/subsystem/LocalDescriptions.properties:27]: Might be indeed a good idea to fix the description as well. I'll pass this to frainone as I'm waiting for her approval of these changes. Thank you @bstansberry

**Review by fl4via — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @petrberan

**Pull request merged**
